src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -1027,11 +1026,11 @@ def __import_kvset_from_file(client, path, yes):
     if KVSetConstants.KVSETRootElementName not in new_kvset:
         raise FileOperationError("file '{0}' is not in a valid '{1}' format.".format(path, ImportExportProfiles.KVSET))
 
-    kvset_to_import = [ConfigurationSetting(key=kv['key'],
-                                            label=kv['label'],
-                                            content_type=kv['content_type'],
-                                            value=kv['value'],
-                                            tags=kv['tags'])
+    kvset_to_import = [ConfigurationSetting(key=kv['key'] if 'key' in kv else None,
```

Comment:
>None [](http://example.com/codeflow?start=78&length=4)

Key should never be null.

Reply:
We have that validation down in `validate_import_key` function with a proper error message.
This default value is here just so that the command won't fail with not so helpful error message.

Reply:
use dict.get(key[, default]) to assign default values.